postman/Get Clicks.postman_collection.json

```diff
@@ -0,0 +1,596 @@
+{
+	"info": {
+		"_postman_id": "c35a94f9-f5a1-4dae-b976-07c9fd4862aa",
+		"name": "Get Clicks",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "Get Details",
+			"item": [
+				{
+					"name": "Get Account",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"id": "5d0f8297-d924-4b24-b51f-f2c4af5ffd5f",
+								"exec": [
+									"pm.test(\"Check consumer_key is defined and not empty\", function(){",
+									"   pm.expect(pm.environment.has(\"consumer_key\")).to.equal(true) && pm.expect(pm.environment.get(\"consumer_key\")).not.eql('');",
+									"});",
+									"",
+									"pm.test(\"Check consumer_secret is defined and not empty\", function(){",
+									"   pm.expect(pm.environment.has(\"consumer_secret\")).to.eql(true) && pm.expect(pm.environment.get(\"consumer_secret\")).not.eql('');",
+									"});",
+									"",
+									"pm.test(\"Check access_token is defined and not empty\", function(){",
+									"   pm.expect(pm.environment.has(\"access_token\")).to.eql(true) && pm.expect(pm.environment.get(\"access_token\")).not.eql('');",
+									"});",
+									"",
+									"pm.test(\"Check token_secret is defined and not empty\", function(){",
+									"   pm.expect(pm.environment.has(\"token_secret\")).to.eql(true) && pm.expect(pm.environment.get(\"token_secret\")).not.eql('');",
+									"});",
+									"",
+									"pm.test(\"Successfully get an account Id\", function(){",
+									"    pm.response.to.have.status(200);",
+									"    var jsonData = pm.response.json();",
+									"    pm.expect(jsonData.entries.length).to.be.above(0);",
+									"    pm.globals.set(\"accountId\",jsonData.entries[0].id ) ;   ",
+									"});",
+									""
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "GET",
+						"header": [],
+						"body": {
+							"mode": "raw",
+							"raw": ""
+						},
+						"url": {
+							"raw": "{{base_url}}/accounts",
+							"host": [
+								"{{base_url}}"
+							],
+							"path": [
+								"accounts"
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "Get Lists on Account",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"id": "4d22c8fa-8695-4098-a275-10658e8aaea5",
+								"exec": [
+									"pm.test(\"Successfully got a  random list\", function() {",
+									"    var jsonData = pm.response.json();",
+									"    pm.expect(jsonData.total_size).to.be.a('number')",
+									"    var list_entry = Math.floor((Math.random() * jsonData.total_size ))",
```

Comment:
Only the first 100 lists are included in this page, while total_size shows the total number of lists on the account. This should instead pick a random number between 0 and the size of the entries array.

Reply:
Updated